client/client.go

```diff
@@ -531,7 +531,7 @@ func (c *authenticatingClient) doAuthenticate() error {
 
 	c.regionServiceURLs = authDetails.RegionServiceURLs
 	if err := c.createServiceURLs(); err != nil {
-		return gooseerrors.Newf(err, "cannot create service URLs")
+		return gooseerrors.Newf(err,"cannot create URL")
```

Comment:
There's a go fmt error here (missing space).
Also, it should be URLs.
If you revert to the original code both issues will be fixed.